core/src/main/scala/caesura/csv.scala

```diff
@@ -19,57 +19,79 @@ package caesura
   the License.
 
 */
+
 import magnolia._
 
 import scala.annotation._
 import scala.language.experimental.macros
 
-object Csv {
- 
+trait Format {
+
+  implicit val stringEncoder: Encoder[String] = s => Row(s)
```

Comment:
I am not sure yet why these instances have to be at the top level. If they were in the Encoder/Decoder object, I was getting the StackOverflowError.